### ISSUE TITLE
Index error while attempting to use getAccounts

### ISSUE THREAD

**Issue by jsgv** [coinbase/coinbase-php#145]
I am getting an `Undefined index: pagination` while using the `$client->getAccounts();` method.
It is correct that the pagination key is not set, but data is all there.
It sometimes works without error. 

![error](https://user-images.githubusercontent.com/2798097/34756027-c705fea6-f57c-11e7-880e-755b69e4481a.png)

Response data
![response](https://user-images.githubusercontent.com/2798097/34756032-cf859438-f57c-11e7-8e37-822b8bcd651a.png)

**Comment by ThatCheck** [comment]
Same error. Do you found a workaround ?

**Comment by jsgv** [comment]
@ThatCheck Yes

Update https://github.com/coinbase/coinbase-php/blob/master/src/Mapper.php#L489-L492

To this:
``` php
if (isset($data['pagination'])) {
    $coll = new ResourceCollection(
        $data['pagination']['previous_uri'],
        $data['pagination']['next_uri']
    );
} else {
    $coll = new ResourceCollection('','');
}
```

**Comment by anhchienhoang** [comment]
Same for me.
@jsgv  don't need to pass empty strings since they're null by default ;)

**Comment by sds** [comment]
Thanks for the report, @jsgv! 

Would you consider submitting a pull request with the fix? Want to ensure you get credit for the solution. Thanks!

**Comment by jsgv** [comment]
@sds PR created.

**Comment by sds** [comment]
Fixed in #145.
